ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -604,6 +606,58 @@ objects:
         else
             echo "Unsupported cluster type '${CLUSTER_TYPE}'"
             exit 1
+        elif [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then
+            base_domain="${BASE_DOMAIN:-origin-ci-int-aws.dev.rhcloud.com}"
+            vsphere_user=$(grep -oP 'vsphere_user="\K[^"]+' ${VSPHERE_VARS_FILE})
+            vsphere_password=$(grep -oP 'vsphere_password="\K[^"]+' ${VSPHERE_VARS_FILE})
+            ipam=$(grep -oP 'ipam="\K[^"]+' ${VSPHERE_VARS_FILE})
+            ipam_token=$(grep -oP 'ipam_token="\K[^"]+' ${VSPHERE_VARS_FILE})
+            declare -a vips
+
+            for i in {0..2}
+            do
+              args = '{"network":"139.178.89.192","hostname":"$CLUSTER_NAME-$i","ipam":"$ipam","ipam_token":"$ipam_token"}'
```

Comment:
@jcpowermac do you think this is the right value for network?

Reply:
See:

https://github.com/openshift/installer/blob/58ecac34ae25797f62fb10504982a8b4562fdaf7/upi/vsphere/machine/ip.tf#L31

And

https://github.com/openshift/release/blob/bd3adfebeac6d3221626a294f8e09d4770f3e6f1/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L1777

Reply:
better? I think I was thrown by this: https://github.com/openshift/installer/blob/master/upi/vsphere/machine/cidr_to_ip.sh#L6

Reply:
I think it needs just to be the ip based on:

https://github.com/openshift/installer/blob/58ecac34ae25797f62fb10504982a8b4562fdaf7/upi/vsphere/machine/ip.tf#L2
https://www.terraform.io/docs/configuration/functions/cidrhost.html